src/tcgwars/logic/util/PokemonCardSet.java

```diff
@@ -285,6 +285,8 @@ public String getName() {
     }
   }
 
+  public String getWholeName() { return getTopPokemonCard().getWholeName(); }
```

Comment:
For the method right above this one, maybe it could be simplified into using `getName()` since that'd make the replace already?

Reply:
Also not sure why that try catch is exactly needed, but if it is would it also be needed in the Card implementation of `getName()`?

Reply:
I've removed the extra string stripping from the `getName` method above this one and also have a try/except block in `getWholeName`